### DISCUSSION TITLE
Please add available Projects to GitHub dashboard

### DISCUSSION THREAD

**Discussion by retiutut** [community/community#54977] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

On the left of my dashboard, I am used to seeing:
- Top repositories
- Recent activities
- Your (my) teams

I am consistently having to go through multiple clicks to quickly access my core team's project Kanban board. While I could make a bookmark in my browser, I currently do not need to bookmark any sites or display a bookmark toolbar in my browser.

Please just add this to the GitHub dashboard/home page.

Plenty of devs and engineers start their day by looking at sprint planning and boards, so I think this would be a useful addition.

Thanks,
RW

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by rileybroughten** [comment]
👋 Thank you for your feedback! This is on our radar 👀
